### DISCUSSION TITLE
ANTs报错

### DISCUSSION THREAD

**Discussion by GuofengHuang** [ANTsX/ANTs#1524] [General]
![8a9bf98022ea3b3fdd2a0077450d1aa](https://user-images.githubusercontent.com/48087971/232288480-0945e9d3-da24-4ce7-af59-2da999fe2bcb.png)
ANTs
cmd: antsRegistrationSyN.sh -d 3 -o /nhkj/run/ants_ -f /nhkj/run/TemplataFlow/tpl-Whs/tpl-Whs_res-1_atlas-v3_desc-brain_mask.nii.gz -m /nhkj/run/data1/rawmridata/sub-rat001/anat/sub-rat001_T1w.nii
Error!!!
I don't know that what can i do

**Comment by stnava** [comment]
you are probably running out of RAM on your system - try antsRegistrationSyNQuick.sh ... if that also fails, you will need to downsample data or further experiment with registration parameters.

**Reply by GuofengHuang** [reply]
Thank you very much for your reply. I tried antsRegistrationSyNQuick.sh and also reported the same error. May I ask if downsample data refers to keeping the original dimensions of the data unchanged and expanding the voxel multiple? Is the target of downsample data Atlas or T2w image?麻烦您了。

**Reply by cookpa** [reply]
Downsample means increase the voxel size, so that the image has fewer voxels. You can use the `ResampleImageBySpacing` command to do this. You might need to do it on both images.

**Comment by GuofengHuang** [comment]
Hello, when I use antsRegistrationSyNQuick.sh -d 3 -f corr_T1_brain.nii.gz -m corr_mag_brain.nii.gz -o magtoT1, it shows that the registration is running but no files are generated. Do you know why this could happen? Thank you!!!


黄国峰
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ANTsX/ANTs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月16日(星期天) 下午5:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ANTsX/ANTs] ANTs报错 (Discussion #1524)





 
you are probably running out of RAM on your system - try antsRegistrationSyNQuick.sh ... if that also fails, you will need to downsample data or further experiment with registration parameters.

**Comment by ntustison** [comment]
Did the registration process complete?  If so, can you reproduce the entire screen output so that we can diagnose the problem?

**Comment by GuofengHuang** [comment]
您好，以下是运行截图，麻烦您给看一下，谢谢啦








黄国峰
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ANTsX/ANTs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月13日(星期六) 晚上10:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ANTsX/ANTs] ANTs报错 (Discussion #1524)





 
Did the registration process complete?  If so, can you reproduce the entire screen output so that we can diagnose the problem?

**Comment by GuofengHuang** [comment]
Hello, attached is a screenshot of the terminal from my execution, and the attachment is a copy of the terminal information. Could you please take a look? Thank you!



黄国峰
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ANTsX/ANTs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月13日(星期六) 晚上10:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ANTsX/ANTs] ANTs报错 (Discussion #1524)





 
Did the registration process complete?  If so, can you reproduce the entire screen output so that we can diagnose the problem?

**Reply by ntustison** [reply]
See above at @cookpa 's response---try downsampling the images before running.
